test/Microsoft.OpenAPI.OData.Reader.Tests/Generator/OpenApiResponseGeneratorTests.cs

```diff
@@ -169,19 +157,7 @@ public void CreateResponseForOperationThrowArgumentNullOperation()
             ODataContext context = new ODataContext(EdmCoreModel.Instance);
 
             // Act & Assert
-            Assert.Throws<ArgumentNullException>("operation", () => context.CreateResponses(operation: null, path: null));
```

Comment:
Why was this test deleted?

Reply:
I removed `path` parameter from `CreateResponses` extension method, so no need to check if the argument is null.